src/charm.py

```diff
@@ -262,7 +289,18 @@ def _wp_generate_password(length=64):
         wp_secrets["default_admin_password"] = secrets.token_urlsafe(32)
         return wp_secrets
 
-    def _replica_relation_data(self):
+    def _replica_relation_data(self) -> RelationDataContent:
+        """Retrieve data shared with WordPress peers(replicas).
```

Comment:
Sometimes the docstrings say "wordpress" and others,  "WordPress"

Reply:
changed all docstrings wordpress(lowercase) to WordPress!